### PR TITLE
Use early returns

### PR DESCRIPTION
**Pull request by dantleech**
This PR updates the DataMapper class to use early returns to make the code more comprehensible.

### CODE REVIEW

File: Form/DataMapper.php [hunk 1]
```diff
@@ -86,39 +86,43 @@ public function mapFormsToData($forms, &$data)
...
+                /** @var $data \Netgen\Bundle\EzFormsBundle\Form\DataWrapper */
+                $this->mapFromForm($form, $data, $propertyPath);
+                continue;
+            }
+
+            // If the field is of type DateTime and the data is the same skip the update to
+            // keep the original object hash
+            if (
+                $form->getData() instanceof \DateTime &&
+                $form->getData() == $this->propertyAccessor->getValue($data, $propertyPath)
```
Comment (dantleech) [inline Form/DataMapper.php]: should be === ?

Comment (dantleech) [inline Form/DataMapper.php]: nope

**Comment by pspanja** [comment]
Thanks @dantleech, +1

**Comment by MarioBlazek** [comment]
Thanks @dantleech.

**Pull request merged**